.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 7
```

Comment:
I feel this time is a bit too short? It's possible people are just on vacation for two week and they intend to get back to the issue afterwards.

Reply:
Thank you @meteorcloudy , If we do not receive reply from user in 7 days we apply stale label and wait another 7 days to close the issue(Total 14 days) , Here is an example [issue](https://github.com/tensorflow/tensorflow/issues/57633).
We can definitely customize this value , please suggest based on your requirement.

Reply:
I personally would prefer a longer time frame, perhaps 14 + 14 days instead of 7 + 7?

Reply:
14 + 14 sounds good to me, let's start with this